pkg/synopsysctl/cmd_root.go

```diff
@@ -53,7 +53,9 @@ var rootCmd = &cobra.Command{
 		}
 		log.SetLevel(lvl)
 		// Sets kubeconfig and initializes resource client libraries
-		setResourceClients()
+		if err := setResourceClients(); err != nil {
+			return err
```

Comment:
Returning an error here will print the `synopsysctl --help` message. Consider logging an error and returning `nil`.   Also, returning an error will prevent synopsysctl from running if you are not connected to a cluster and there is an error from getting kubeconfig/kubeclient.